### ISSUE TITLE
Cannot install on Pop!_OS 21.04

### ISSUE THREAD

**Issue by tunix** [flathub/com.brave.Browser#89]
![image](https://user-images.githubusercontent.com/47731/142278070-4c6d0694-a55c-424f-8f72-30380ae6a868.png)

![image](https://user-images.githubusercontent.com/47731/142278044-0a28a488-7e43-402b-a178-ef7a36af7206.png)

![image](https://user-images.githubusercontent.com/47731/142278098-e68eb1b8-8053-4f40-a730-33869842365f.png)

![image](https://user-images.githubusercontent.com/47731/142278130-b78e2b0d-9a19-434a-acb4-dc295850e1fe.png)

**Comment by rany2** [comment]
This is because Flathub Beta is installed as a system repository which means it doesn't have access to the runtimes provided by Flathub Stable which is a user repository..

To fix, remove Flathub Beta from being a system repository and readd it as a user repository and it will work.

tl;dr version:

```
sudo flatpak remote-delete --system flathub-beta
flatpak remote-add --user flathub-beta https://flathub.org/beta-repo/flathub-beta.flatpakrepo
flatpak install --user flathub-beta com.brave.Browser
```
